webapp/sources/rudder/rudder-web/src/main/elm/build-app.sh

```diff
@@ -1,11 +1,26 @@
 #!/bin/bash
 
+set -e
+
+if ! command -v elm-0.19.1 &> /dev/null
+then
+  echo "# ERROR: missing elm-0.19.1 binary"
+  echo "# To install the right compiler version:"
+  echo ""
+  echo "$ curl -L -o elm-0.19.1.gz https://github.com/elm/compiler/releases/download/0.19.1/binary-for-linux-64-bit.gz"
+  echo "$ gzip -d elm-0.19.1.gz"
+  echo "$ chmod +x elm-0.19.1"
+  echo "# then put it somewhere in your PATH"
+  exit
+fi
+
 # we want that all elm-stuff stay in src/main/elm
 # whatever the path from which this script is called
 ELM_DIR="$( cd "$( dirname "$0" )" && pwd )"
-PROJECTS=("notifications" "healthcheck" "editor" "onboarding" "rules")
+PROJECTS=("Notifications" "Healthcheck" "Editor" "Onboarding" "Rules")
 for PROJECT in ${PROJECTS[*]}; do
-  cd ${ELM_DIR}/${PROJECT}
-  elm make --optimize sources/rudder-${PROJECT}.elm --output=generated/rudder-${PROJECT}.js
```

Comment:
I don't know how it could work ad rudder-notifications.elm was not in a sources directory so maybe I'm misunderstanding something